tools/internal_ci/linux/grpc_xds_php_test_in_docker.sh

```diff
@@ -93,4 +93,4 @@ GRPC_VERBOSITY=debug GRPC_TRACE=xds_client,xds_resolver,xds_cluster_manager_lb,c
   --gcp_suffix=$(date '+%s') \
   --verbose \
   ${XDS_V3_OPT-} \
-  --client_cmd='./src/php/bin/run_xds_client.sh --server=xds:///{server_uri} --stats_port={stats_port} --qps={qps} {fail_on_failed_rpc} {rpcs_to_send} {metadata_to_send}'
+  --client_cmd='php -d extension=grpc.so -d extension=pthreads.so src/php/tests/interop/xds_client.php --server=xds:///{server_uri} --stats_port={stats_port} --qps={qps} {fail_on_failed_rpc} {rpcs_to_send} {metadata_to_send}'
```

Comment:
To reviewers: this is me trying to tag along to debug some mysterious SIGTERM causing some flakes in other PHP xds interop tests. Can safely ignore this for now. This has nothing to do with the `fault_injection` test we are trying to add in this PR.

Reply:
If possible, let's not check-in debugging code. You can run xDS interop tests on GCE with simple steps: https://menghanl.users.x20web.corp.google.com/www/xds_interop_test_prod_personal_notes.html.

Reply:
This is actually needed in the short term. A prior change (#25696) is causing some flakes on `master`. Switching this back will stop the flakes. I still need to investigate the underlying root cause separately (i.e. why I am observing some unexplained SIGTERM). But this will help the test stability.

Reply:
Testdriver sends SIGTERM after each case to kill and re-create the testing client. If the test case is passing, then it's fine. If the test case is failing, there might be other hidden errors somewhere.

Reply:
Yes that is the expected SIGTERM after each test case to kill the PHP client. Those were being handled properly. But I am seeing some flakes (especially, somehow, only on the `round_robin` and `ping_pong` tests) that I am seeing SIGTERM much earlier during the test before the test passed, when I don't expect it.

Reply:
A flake looks like this: https://fusion.corp.googleusercontent.com/file/autoview?fileUri=googlefile%3A%2Fbigstore%2Fgrpc-testing-kokoro-prod%2Ftest_result_public%2Fprod%2Fgrpc%2Fcore%2Fmaster%2Flinux%2Fgrpc_xds_v3_php%2F308%2F20210409-062404%2Fgithub%2Fgrpc%2Freports%2Fping_pong%2Fsponge_log.log&filename=test.log&invocationId=d28ddda5-34ae-4a98-8b1a-e65a5ee9036d&actionName=invocations%2Fd28ddda5-34ae-4a98-8b1a-e65a5ee9036d%2Ftargets%2Fgithub%252Fgrpc%252Freports%252Fping_pong%2FconfiguredTargets%2Fdefault%2Factions%2Ftest_attempt0&contentType=ansi

The SIGTERM handler you see at the end, is the one I expect to receive after each test case is finished, so that I can kill my PHP client. But in this flake, I am receiving that SIGTERM much earlier than I expected.

Reply:
If the client received multiple SIGTERM, it does sound like a bug.